cfdtools/hdf5/_hdf5.py

```diff
@@ -30,8 +30,7 @@ def open(self, mode='r', datatype=None):
         try:
             self._h5file = h5py.File(self._path, mode=mode)
         except NameError:
-            io.print('error', "h5py could not be imported")
```

Comment:
For code on previous lines:
Delete lines 16, 11, 6
Remove error_stop (unused)
25 files use `import cfdtools.api as api`
2 files use `from cfdtools.api import io, ...` (this one and `cfdtools/cgns/cgns.py`)
Standardize ?

Reply:
`io` was sufficient to use print. The evolution of some messages to error_stop makes `api` necessary. That's not a concern for me (just to avoid both in the same).

Reply:
Lines 16 and 6 are imposed by the Black code formatter.
Ok for line 11.
I will also move line 12 inside the try statement.
error_stop is used.
Question about standardization remains for another dev.